src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayload.cs

```diff
@@ -149,7 +149,7 @@ public virtual bool TryGetBlock(out Block? block, UInt256? totalDifficulty = nul
                 WithdrawalsRoot = Withdrawals is null ? null : new WithdrawalTrie(Withdrawals).RootHash,
             };
 
-            block = new(header, transactions, Array.Empty<BlockHeader>(), Withdrawals);
+            block = new(header, transactions, Array.Empty<BlockHeader>(), Withdrawals, Array.Empty<Transaction>());
```

Comment:
Should be null here. Generally you shouldn't set it as empty list, it's null by default